balta/src/main/scala/za/co/absa/db/balta/classes/DBFunction.scala

```diff
@@ -45,6 +45,24 @@ sealed abstract class DBFunction private(functionName: String,
     s"SELECT * FROM $functionName($paramsLine) $orderBy"
   }
 
+  /**
+   * Executes the function without caring about the result. The goal is the side-effect of the function.
+   * @param connection  - the database connection
+   */
+  def perform()(implicit connection: DBConnection): Unit = {
+    execute("")(_ => ())
+  }
+
+  /**
+   * Executes the function without any verification procedure. It instantiates the function result(s) and returns them in
+   * a list.
+   * @param orderBy     - the clause how to order the function result, if empty, default ordering is preserved
+   * @param connection  - the database connection
+   */
+  def getResult(orderBy: String = "")(implicit connection: DBConnection): List[QueryResultRow] = {
```

Comment:
TIny

The `orderBy` 'API' is different in this function and in the `execute` function. That I find a little bit confusing from users' point of view. Perhaps as they are sharing similar ideas should have similar APIs.

So I would consider making it default also in `def execute[R](orderBy: String)` (such change is backward compatible and also doesn't overwhelm the user thus probably more preferred variant).

Reply:
Also I would document / provide an example of `orderBy` content, because it's not the full clause (`ORDER BY` is already provided)

Reply:
discussed on a call